common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -109,7 +108,7 @@
                 this.sidebar_padding = 10;
                 this.boardName = null;
                 this.current_search = '';
-                this.mode = 'all';
+                this.mode = options.mode || 'commentables';
```

Comment:
Inline sorting and Board view topic selection is using 'commentables', while the others: team + Board all discussion uses 'all'. With sorting thread, it changes the mode.

The profile page sorting is fixed. Now the `Unread` in profile page discussion filtering doesn't really make sense, due to the Unread state as we know.